akka-cluster-metrics/src/test/scala/akka/cluster/metrics/AbstractSigarProxy.scala

```diff
@@ -0,0 +1,3 @@
+package akka.cluster.metrics class AbstractSigarProxy {
```

Comment:
is this used?

Reply:
oops, leftover from an attempt to also remove it from cluster-metrics tests